packages/auth/src/authClients/goTrue.ts

```diff
@@ -33,8 +33,12 @@ export const goTrue = (client: GoTrue): AuthClientGoTrue => {
     signup: async ({ email, password, remember }) =>
       client.signup(email, password, remember),
     getToken: async () => {
-      const user = await client.currentUser()
-      return user?.jwt() || null
+      try {
+        const user = await client.currentUser()
+        return user?.jwt() || null
+      } catch {
```

Comment:
Open to alternatives — I didn’t check what happens if an error is raised but wasn’t that the issue? An error here broke cell data fetching?

Reply:
I left my comment open ended because I couldn't think of a good alternative. Was hoping you could come up with something 🙃 

Let's merge this. Then revisit when the netlify people have answered on my PR.

Reply:
😀